README.md

```diff
@@ -1,16 +1,19 @@
 Developer guide to Tessel 2 Docs Gitbook
 ============
 
-# Tools
-Install the following command line tools using `npm i {{toolname}} -g`
+# Requirements
+The docs are build using a command line tool called [GitBook](https://www.npmjs.com/package/gitbook), which requires:
 
-* [gitbook](https://www.npmjs.com/package/gitbook)
-* [gitbook-summary](https://github.com/imfly/gitbook-summary)
+* [Node Version 4 or higher](https://nodejs.org/en/)
+* [NPM Version 2 or higher](https://docs.npmjs.com/getting-started/installing-node)
 
 # Running locally
 
-Clone, `npm install`, and `gitbook serve`
+* `git clone https://github.com/tessel/t2-docs.git`
```

Comment:
Better to let people choose their own https/ssh preference. I assume this change was made for simpler accessibility, but if so it's a mysterious line of code. Better to link out to github's instructions on cloning.